pygame.py

```diff
@@ -0,0 +1,67 @@
+# Needed to use any and all python resources.
+import pygame
+import pygame_menu  #pip install pygame-menu
+from os import path
+from win32api import GetSystemMetrics 
+import yahtzee        
+
+# Defines common colors
+background_one = (255, 255, 255)  # white
```

Comment:
Colours and window height, width is something I also incorporated into a class with basic game settings. That's simply personal preference however! Your way works perfect!